pyanaconda/kickstart.py

```diff
@@ -289,21 +289,6 @@ def __str__(self):
         storage_module_proxy = STORAGE.get_proxy()
         return storage_module_proxy.GenerateKickstart()
 
-class Firewall(RemovedCommand):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.packages = []
-
-    def __str__(self):
-        # The kickstart for this command is generated by the Firewall sub module
-        return ""
-
-    def setup(self):
-        firewall_proxy = NETWORK.get_proxy(FIREWALL)
-        if firewall_proxy.FirewallKickstarted:
```

Comment:
It looks like you can remove this DBus method.

Reply:
Indeed! Dropped. :)